game/recastnavigation_bin.ksy

```diff
@@ -0,0 +1,283 @@
+meta:
+  id: recastnavigation_bin
+  file-extension: bin
+  license: CC0-1.0
+  endian: le
+doc: this spec can be used to parse recastnavigation binary files
```

Comment:
It's obvious.

Reply:
It's not resolved. Instead of describing that the file is a spec (it is always a spec) and that it is for parsing (KS specs are always for parsing) recastnavigation binary files (the title and `id` says about it) you should describe what the format is for, why is it needed and how to obtain the files of this format.